C_CPP/Templates/.gitignore

```diff
@@ -0,0 +1,32 @@
+# Prerequisites
+*.d
+
+# Compiled Object files
```

Comment:
please add comments explaining why and where these formats are used.

Reply:
I have added comments Why and where these formats are used.